### ISSUE TITLE
Slideshow causes Siteimprove Error "Aria-atomic='true' missing on error container"

### ISSUE THREAD

**Issue by mmafe** [ccswbs/hjckrrh#882]
Siteimprove is assuming the play button on the slideshow is associated with an error and is recommending the addition of ```aria-atomic=true``` so that assistive tech will read out the full error. 

As this is not related to an error container, technique ARIA 19 does not apply. 

Would recommend doing extensive screen-reader testing if we try to get rid of this Siteimprove Error (e.g. it can become very irritating if your screen-reader is constantly reading out the full slideshow text every time it changes).
